qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -3,6 +3,7 @@
 
 from qcodes import VisaInstrument, InstrumentChannel
 from qcodes.utils.validators import Numbers, Bool, Enum, Ints
+from qcodes import DelegateParameter
```

Comment:
Please use the full path for DelegateParameter.

We try to not use the shortcuts within qcodes as it can result in circular dependencies. 
I think the right import is 
`
from qcodes.instruments.parameter import DelegateParameter
`

Reply:
For bonus points fix the imports of VisaInstrument and InsturmentChannel above